java-manta-client-unshaded/src/test/java/com/joyent/manta/client/crypto/MultiCryptoProviderDecryptionTest.java

```diff
@@ -151,7 +152,7 @@ private void validateEncryptedFileWithHeaderValues(final SupportedCipherDetails
 
     private void populateHeaderValuesFromResources(final MantaHttpHeaders getResponseHttpHeaders,
                                                    final SupportedCipherDetails cipherDetails) throws IOException {
-        final String cipherName = cipherDetails.getCipherId().replaceAll("/","-");
+        final String cipherName = cipherDetails.getCipherId().replaceAll(SEPARATOR,"-");
```

Comment:
Same comment above. The cipher separator isn't the same as the directory separator.

Reply:
Noted.